packages/tasks/src/hardware.ts

```diff
@@ -152,6 +152,14 @@ export const SKUS = {
 				tflops: 43.94,
 				memory: [16],
 			},
+			"RTX 5060 Ti": {
+				tflops: 23.70, // source https://www.techpowerup.com/gpu-specs/geforce-rtx-5060-ti.c4246
+				memory: [16],
```

Comment:
```suggestion
				memory: [16, 8],
```

There are variants with 8 and 16 GB, I think.